components/LoginModal.vue

```diff
@@ -317,6 +322,12 @@ export default {
   border-radius: var(--spacer-2xs);
   padding: var(--spacer-xs) calc(var(--spacer-base) * 2.8) 0 calc(var(--spacer-base) * 2.8);
   align-self: center;
+
+  &__is-disabled--button {
+    --button-background: var(--_c-green-primary-disabled);
+
+    color: var(--_c-white-disabled);
+  }
```

Comment:
I think manually no need to include css here. it can be handled on component itself pls add this in your sfButton -- class="color-primary"

Reply:
Resolved